.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: go
 
 go:
-  - "1.9"
-  - "1.10"
+  - "1.15"
+  - "1.16"
```

Comment:
Can we go to: golang 1.16.6, golang 1.15.14, there's a new CVE in versions less than that.  1.17 is also free of issues CVE wise, but I'm not sure if it's available everywhere yet.

Reply:
"1.15.x" and "1.16.x" should get us what we want.  1.17 is not yet released.